README.md

```diff
@@ -108,27 +108,22 @@ psql -h 127.0.0.1 -p 15432 -U postgres
 
 We have integrated a setup tool in the Docker image that helps replicate data from your primary (MySQL|Postgres) server to MyDuck Server. The tool is available via the `SETUP_MODE` environment variable. In `REPLICA` mode, the container will start MyDuck Server, dump a snapshot of your primary (MySQL|Postgres) server, and start replicating data in real-time.
 
-#### MySQL Replica Setup
+#### MySQL or Postgres Replica Setup
```

Comment:
This subsection title could be removed now.

Reply:
fixed